community/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+mod social_db;
+
+use crate::social_db::{ext_social_db, SOCIAL_DB};
+use near_sdk::borsh::{BorshDeserialize, BorshSerialize};
+use near_sdk::{env, near_bindgen, require, AccountId, NearToken, Promise};
+
+const DEVHUB: &str = "devgovgigs.near";
```

Comment:
You can now do:

```suggestion
const DEVHUB: &near_sdk::AccountIdRef = near_sdk::AccountIdRef::new_or_panic("devgovgigs.near");
```

and then you won't need to do `parse` at runtime.

Reply:
Nice!